pkg/instances/instances.go

```diff
@@ -88,6 +90,47 @@ func (i *instances) Logs(id string) string {
 	return ""
 }
 
+func (i *instances) YAML(id string) string {
```

Comment:
Please change the method name to GetConfiguration(... ) ..and add comment that it returns the YAML configuration..